UnityProject/Assets/Scripts/UI/Core/Net/Elements/Dynamic/Spawned/DNAStrandList.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using Objects.Machines;
+using Objects.Medical;
+using UI.Core.NetUI;
+using UI.Objects.Cargo;
+using UI.Objects.Medical;
+using UI.Objects.Medical.Cloning;
+using UnityEngine;
+
+namespace UI.Core.Net.Elements.Dynamic.Spawned
+{
+	public class DNAStrandList : EmptyItemList
+	{
+
+
+		public DNAStrandElement AddElement(DNAMutationData.DNAPayload Payload, string target, DNAStrandElement.Location SetLocation)
+		{
+			var NewElement  = AddItem() as DNAStrandElement;
```

Comment:
I don't get this code, what does it do?

Reply:
me trying to work out how Netlists work, turns out I don't need to have it, but already made it so Can't be bothered to move it,you think it's worth saying adding a comment that an empty list would just work is fine?